spec/features/map_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Create Map object', type: :feature do
```

Comment:
I know there is a pattern of testing like this, but to me, each type of object is not a feature and doesn't deserve a (slow) feature test.  The feature is "Create an object", all the variation could be in faster more focused unit tests.

Reply:
OK, removing this test, will add to `digital_object_spec.rb`